distro/logger.go

```diff
@@ -51,7 +53,7 @@ var (
 // zapLevel returns the parsed zapcore.Level.
 func zapLevel(level string) zapcore.Level {
 	for _, l := range logLevels {
-		if l.String() == level {
+		if l.String() == strings.ToLower(level) {
```

Comment:
No need to call this every loop iteration.

```suggestion
	level = strings.ToLower(level)
	for _, l := range logLevels {
		if l.String() == level {
```